README.md

```diff
@@ -69,19 +69,42 @@ to modify the build scripts. For example, to disable the local build cache when
 <details>
   <summary>Click to see the complete set of available system properties and environment variables in the table below. </summary>
 
-| Gradle Enterprise / Build Cache API | System property | Environment variable |
-| :-------------- | :-------------  | :------------------- |
-|gradleEnterprise.server|gradle.enterprise.url|GRADLE_ENTERPRISE_URL|
-|gradleEnterprise.allowUntrustedServer|gradle.enterprise.allowUntrustedServer|GRADLE_ENTERPRISE_ALLOWUNTRUSTEDSERVER|
-|buildCache.local.setEnabled|gradle.cache.local.enabled|GRADLE_CACHE_LOCAL_ENABLED|
-|buildCache.local.setPush|gradle.cache.local.push|GRADLE_CACHE_LOCAL_PUSH|
-|buildCache.local.setDirectory|gradle.cache.local.directory|GRADLE_CACHE_LOCAL_DIRECTORY|
-|buildCache.local.setRemoveUnusedEntriesAfterDays|gradle.cache.local.removeUnusedEntriesAfterDays|GRADLE_CACHE_LOCAL_REMOVEUNUSEDENTRIESAFTERDAYS|
-|buildCache.remote.setEnabled|gradle.cache.remote.enabled|GRADLE_CACHE_REMOTE_ENABLED|
-|buildCache.remote.setPush|gradle.cache.remote.push|GRADLE_CACHE_REMOTE_PUSH|
-|buildCache.remote.setUrl|gradle.cache.remote.url|GRADLE_CACHE_REMOTE_URL|
-|buildCache.remote.setAllowUntrustedServer|gradle.cache.remote.allowUntrustedServer|GRADLE_CACHE_REMOTE_ALLOWUNTRUSTEDSERVER|
-|---|gradle.cache.remote.shard|GRADLE_CACHE_REMOTE_SHARD|
+### Gradle Enterprise Settings
```

Comment:
I broke this table apart, as the value in the original `Gradle Enterprise / Build Cache API` column could now be conditional based on whether the remote cache was HTTP or the Gradle Enterprise connector. This was true for the url and shard overrides. 

I'm certainly open to suggestions on how to present this information more clearly / succinctly.

Reply:
Good idea. I made some more changes on top of yours.